### ISSUE TITLE
Admin 6.2.6 cannot start

### ISSUE THREAD

**Issue by drhideg** [ioBroker/ioBroker.admin#1616]
Admin 6.2.6 cannot be started until I remove the question mark.

```
2022-07-29 12:21:19.800 - error: host.dell Caught by controller[0]: /opt/iobroker/node_modules/iobroker.admin/lib/web.js:955
2022-07-29 12:21:19.800 - error: host.dell Caught by controller[0]: if (result?.error) {
2022-07-29 12:21:19.800 - error: host.dell Caught by controller[0]: ^
2022-07-29 12:21:19.801 - error: host.dell Caught by controller[0]: SyntaxError: Unexpected token '.'
2022-07-29 12:21:19.801 - error: host.dell Caught by controller[0]: at wrapSafe (internal/modules/cjs/loader.js:915:16)
2022-07-29 12:21:19.801 - error: host.dell Caught by controller[0]: at Module._compile (internal/modules/cjs/loader.js:963:27)
2022-07-29 12:21:19.801 - error: host.dell Caught by controller[0]: at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
2022-07-29 12:21:19.802 - error: host.dell Caught by controller[0]: at Module.load (internal/modules/cjs/loader.js:863:32)
2022-07-29 12:21:19.802 - error: host.dell Caught by controller[0]: at Function.Module._load (internal/modules/cjs/loader.js:708:14)
2022-07-29 12:21:19.802 - error: host.dell Caught by controller[0]: at Module.require (internal/modules/cjs/loader.js:887:19)
2022-07-29 12:21:19.802 - error: host.dell Caught by controller[0]: at require (internal/modules/cjs/helpers.js:74:18)
2022-07-29 12:21:19.803 - error: host.dell Caught by controller[0]: at Object. (/opt/iobroker/node_modules/iobroker.admin/main.js:27:21)
2022-07-29 12:21:19.803 - error: host.dell Caught by controller[0]: at Module._compile (internal/modules/cjs/loader.js:999:30)
2022-07-29 12:21:19.803 - error: host.dell Caught by controller[0]: at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
2022-07-29 12:21:19.803 - error: host.dell instance system.adapter.admin.0 terminated with code 1 (JS_CONTROLLER_STOPPED)
```

**Comment by Feuer-sturm** [comment]
@drhideg Which questionmark did you remove?

**Comment by drhideg** [comment]
> @drhideg Which questionmark did you remove?

@Feuer-sturm lib/web.js line 955
 `if (result?.error) {` changed to `if (result.error) {`

**Comment by Apollon77** [comment]
fixed already in 6.2.9
